Apps/Sandcastle/gallery/HeadingPitchRoll.html

```diff
@@ -208,7 +208,7 @@
     speedVector = Cesium.Cartesian3.multiplyByScalar(Cesium.Cartesian3.UNIT_X, speed / 10, speedVector);
     position = Cesium.Matrix4.multiplyByPoint(planePrimitive.modelMatrix, speedVector, position);
     pathPosition.addSample(Cesium.JulianDate.now(), position);
-    Cesium.Transforms.headingPitchRollToFixedFrame(position, hpRoll, Cesium.Ellipsoid.WGS84, fixedFrameTransform, planePrimitive.modelMatrix);
+    Cesium.Transforms.directHeadingPitchRollToFixedFrame(position, hpRoll, Cesium.Ellipsoid.WGS84, fixedFrameTransform, planePrimitive.modelMatrix);
```

Comment:
`directHeadingPitchRollToFixedFrame` takes the same parameters than `headingPitchRollToFixedFrame`. Internally, it uses `Quaternion.fromDirectHeadingPitchRoll` instead of `Quaternion.fromHeadingPitchRoll`. For now in this PR we have two groups of functions that we shouldn't mix. The first group, that should be deprecated, has the next functions:
 -  `HeadingPitchRoll.fromQuaternion`,
 - `Matrix3.fromHeadingPitchRoll`,
 - `Quaternion.fromHeadingPitchRoll`,
 - `Transforms.HeadingPitchRollToFixedFrame`.

The newcomer group, that should replace the first group, has the next functions:
 -  `HeadingPitchRoll.fromDirectQuaternion`,
 - `Matrix3.fromDirectHeadingPitchRoll`,
 - `Quaternion.fromDirectHeadingPitchRoll`,
 - `Transforms.directHeadingPitchRollToFixedFrame`.

A function in the first group and its equivalent in the second group takes the same parameters in the same orders and return the same type of result (HeadingPitchRoll, Matrix3, Quaternion or Matrix4 instances respectively). The only difference is the sense of rotation for heading and pitch used internally.

Normally, I checked in my IDE that each instance of the first group functions and if needed, I changed for the equivalent in the second group. Also I didn't replace the functions when the parameters where heading=0, pitch=0 because there will be no change in their behaviour.